stress_test.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+"""
+Stress Test Script
+
+This script performs load testing on the eDrop API endpoint by sending concurrent requests
+at a specified rate for a specified duration.
+
+Usage:
+    python stress_test.py --host HOST_URL [options]
+
+Options:
+    --host      Base host URL (required)
+    --rps       Requests per second (default: 20)
+    --duration  Test duration in seconds (default: 60)
+
+Examples:
+    # Run against your eDrop host with default settings (20 rps, 60 seconds)
+    python stress_test.py --host http://your-edrop-api-host/edrop/
+
+    # Run with custom settings
+    python stress_test.py --host http://your-edrop-api-host/edrop/ --rps 30 --duration 120
+
+    # Run a quick test with lower values
+    python stress_test.py --host http://your-edrop-api-host/edrop/ --rps 5 --duration 10
+
+Results:
+    - Prints real-time progress during the test
+    - Shows summary statistics after completion
+    - Saves detailed results to 'load_test_results.json'
+"""
+
+import aiohttp
+import asyncio
+import time
+from datetime import datetime
+import json
+import argparse
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(description='Run stress test for edrop API')
+    parser.add_argument('--host', required=True,
+                      help='Base host URL (required)')
+    parser.add_argument('--rps', type=int, default=20,
+                      help='Requests per second (default: 20)')
+    parser.add_argument('--duration', type=int, default=60,
+                      help='Total duration in seconds (default: 60)')
+    return parser.parse_args()
+
+
+# Test payload for the API requests
+TEST_PAYLOAD = {
+    'instrument': 'consent',
+    'record': '999999', # this is a test record id hence it will not exist in the database
```

Comment:
I think we actually want this to vary, otherwise it'll just bounce right back after the first request, which creates the record.

Reply:
In other words, each request should go through the motions of creating an order and sending it to GBF.